external/vpx-batman66/README.md

```diff
@@ -0,0 +1,37 @@
+# Batman 66 (Stern Tribute Table)
+
+![Table Preview](https://vpuniverse.com/screenshots/monthly_2022_03/b66-1.thumb.png.6df534e4f494fb82182944e7539caf2c.png)
+
+Author: [daphishbowl](https://vpuniverse.com/profile/32679-daphishbowl/)  
+Version: 1.1.0
+Download: [VP Universe](https://vpuniverse.com/files/file/6868-batman-66-stern-tribute/)
+
+DirectB2S
+
+Author: [hauntfreaks](https://vpuniverse.com/profile/5216-hauntfreaks/) 
+Version: 1.0
+Download: [VP Universe] https://vpuniverse.com/files/file/6949-batman-66-stern-2016-b2s/
+
+ROM
+
+Download: Included with table download
+
+PUP Pack
+
+Download [Mega.nz] https://mega.nz/file/XMIgnRwC#LmRlA5GjAzxrWdjxl4O87PxqjUldOF4kA-fLoU0odFQ
+
+Note - PUP Pack was not working at time of testing
+
+## Status 
+
+Minimum VPX Standalone build: 10.8.0-1989-a764013
+| Playfield | Controls | Backglass | DMD | ROM Required | FPS | 
+|-----------|----------|-----------|-----|--------------|-----|
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | 30 |
+
+## Instructions
+
+- Copy the contents of this repo folder to your USB drive
+- Add your personalized launcher.elf and rename it to vpx-nascar.elf
+- Download the table and directb2s versions listed above and copy them into this folder
+- You're off to the races 🏁
```

Comment:
Looks like this was copied from NASCAR. Change the witty instruction.

Reply:
Resolved